specifications/api/swagger.yaml

```diff
@@ -7652,3 +7657,18 @@ paths:
           content:
             application/json:
               schema: *ApiBlockHeader
+
+  /database/status/transactions/cbor:
+    get:
+      tags: ["Utils"]
+      summary: transactions CBOR syncing status
```

Comment:
It would be nice to better describe it, specifically: what is the transactions CBOR syncing status?